src/main/webapp/pluggable/dataset-analisys-results.xhtml

```diff
@@ -0,0 +1,94 @@
+<ui:composition
+        xmlns="http://www.w3.org/1999/xhtml"
+        xmlns:h="http://java.sun.com/jsf/html"
+        xmlns:f="http://java.sun.com/jsf/core"
+        xmlns:ui="http://java.sun.com/jsf/facelets"
+        xmlns:p="http://primefaces.org/ui"
+        xmlns:jsf="http://xmlns.jcp.org/jsf"
+        xmlns:dv="http://dataverse.org/facelets"
+        xmlns:o="http://omnifaces.org/ui">
+
+    <o:importConstants type="pl.edu.icm.pl.mxrdr.extension.importer.xds.XdsAnalysisField" />
+    <f:loadBundle var="mxrdrBundle" basename="MxrdrBundle"/>
+
+    <ui:fragment rendered="#{datasetAnalysisTab.showAnalysisResults()}">
+
+        <h:panelGroup id="analysisContent">
+            <p:dataTable id="analysisTable" class="no-alternating-row-colors"
+                         value="#{XdsAnalysisField.values()}" var="analysisField" widgetVar="analysisTable">
+
+                <p:column headerText="#{bundle['file.dataFilesTab.analysisResults.header.field']}" class="col-sm-3">
+                    <h:outputText value="#{datasetAnalysisTab.getDatasetFieldName(analysisField.getName())}"/>
+                </p:column>
+                <p:column headerText="#{bundle['file.dataFilesTab.analysisResults.header.inMetadata']}" class="col-sm-3">
+                    <h:outputText value="#{datasetAnalysisTab.getDatasetFieldValue(analysisField.getName(), 'PRIMARY')}"/>
+                </p:column>
+                <p:column headerText="#{bundle['file.dataFilesTab.analysisResults.header.inAnalysis']}" class="col-sm-3">
+                    <h:outputText value="#{datasetAnalysisTab.getDatasetFieldValue(analysisField.getName(), 'XDS')}"/>
+                </p:column>
+
+            </p:dataTable>
+            
+            <div class="col-sm-9">
+                <h:outputText value="#{mxrdrBundle['dataset.analysisTab.artifacts.message']}" />
+            </div>
+            <p:dataTable id="artifactsTable"
+                         value="#{datasetAnalysisTab.getArtifacts()}" var="artifact" widgetVar="artifactsTable">
+                         
+                <p:column class="col-sm-9">
+                    
+                    <p:commandLink action="#{datasetAnalysisTab.downloadArtifact(artifact.getName(), artifact.getStorageLocation())}">
+                        <h:outputText value="#{artifact.getName()}" />
+                    </p:commandLink>
+                </p:column>
+            </p:dataTable>
+            
+        </h:panelGroup>
+    </ui:fragment>
+
+    <ui:fragment rendered="#{!datasetAnalysisTab.showAnalysisResults()}">
+        <div class="panel panel-default">
+            <div class="panel-body">
+                <div class="col-sm-9">
+                    <ui:fragment rendered="#{datasetAnalysisTab.isDatasetInDraft()}">
+                        <p>
```

Comment:
If we add `class="text-block"` then we would have better display with 80 character display and character spacing (this can be switched on/off in header)

Reply:
Done